src/plugins/workspace/server/saved_objects/workspace_saved_objects_client_wrapper.ts

```diff
@@ -330,6 +349,9 @@ export class WorkspaceSavedObjectsClientWrapper {
       attributes: T,
       options?: SavedObjectsCreateOptions
     ) => {
+      // Only OSD admin can create workspace
+      if (isTypeContained(WORKSPACE_TYPE, type)) throw generateOSDAdminPermissionError();
```

Comment:
Nit: I'd prefer simply do `if (WORKSPACE_TYPE === type)`

Reply:
Sure, I will simply the code.